dataactcore/migrations/versions/cd1025ac9399_remove_duplicate_columns.py

```diff
@@ -0,0 +1,42 @@
+"""remove duplicate columns
+
+Revision ID: cd1025ac9399
+Revises: a767facf8ea8
+Create Date: 2017-09-18 15:05:52.772251
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'cd1025ac9399'
+down_revision = 'a767facf8ea8'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+
+
+def upgrade_data_broker():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('award_procurement', sa.Column('base_and_all_options_value', sa.Text(), nullable=True))
+    op.add_column('award_procurement', sa.Column('base_exercised_options_val', sa.Text(), nullable=True))
```

Comment:
If we're removing columns why are we doing `add_column` here? I am unclear, please confirm.